src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -687,14 +687,16 @@ void Unit::DealDamageMods(Unit* victim, uint32 &damage, uint32* absorb)
 
     if (sWorld->getBoolConfig(CONFIG_LEGACY_BUFF_ENABLED))
     {
-        // Damage increase with expansion/level difference
-        if (ToPlayer() && victim->ToCreature())
+        int16 levelDiff = getLevel() - victim->getLevelForTarget(this);
+
+        if (levelDiff > 0)
         {
-            int16 levelDiff = getLevel() - victim->getLevelForTarget(this);
+            const uint32 intendedItemLevelByExpansion[CURRENT_EXPANSION] = { 65, 115, 200, 346, 0, 0 };
+            float damageMultiplier = 1.0f;
 
-            if (levelDiff > 0)
+            // Damage increase with expansion/level difference
+            if (ToPlayer() && victim->ToCreature())
```

Comment:
Yup, to be fair i hoped my previous PR would be merged (https://github.com/TrinityCore/TrinityCore/pull/19264/files) so i can use "IsPlayer()" directly x)